.github/CODEOWNERS

```diff
@@ -267,8 +267,11 @@
 /core/storage/statistics/ @brianrodri
 /core/templates/components/common-layout-directives/common-elements/confirmation-modal.template.html @brianrodri
 /core/templates/domain/statistics/ @brianrodri
+/core/templates/domain/improvements/ @brianrodri
 /core/templates/pages/exploration-editor-page/improvements-tab/ @brianrodri
 /core/templates/pages/exploration-editor-page/statistics-tab/ @brianrodri
+/core/templates/services/exploration-stats.service*.ts @brianrodri
```

Comment:
@brianrodri, should we move this service to `/core/templates/pages/exploration-editor-page/statistics-tab/` if it's only going to be used in the exploration stats tab? (not sure!)

Reply:
Hmm don't think so, it will also be used by the `improvements-tab` so at least one other place will use it. I think it's fine to keep it here for now.